### PR TITLE
docs: release note drafts

### PR DESCRIPTION
**Pull request by davecheney**
Release note drafts for 0.13.0 release.

**DO NOT MERGE**

Copy and paste into the release notes dialog box as required.

Signed-off-by: Dave Cheney <dave@cheney.net>

### CODE REVIEW

File: docs/v0.13.0-release-notes.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+
+Contour 0.13.0 now prioritizes `force-ssl-redirect`. If this annotation is specified and set to `true`, Contour will always register a port 80 route for the ingress, even if `ingress.allow-http: false`, so that the forced upgrade can take effect.
+
+Fixes #1023 with many thanks to @ceralena.
+
+### `Maglev` and `RingHash` load balancer strategies no longer supported.
+
+`RingHash` and `Maglev` are two balancing / affinity strategies offered by IngressRoute. However due to a lack of understanding of how they worked when they were added in Contour 0.6, neither strategy was properly configured and would only result in random behaviour. 
+Without the ability to configure the hash key, which is usually some for of session af session or cookie, these strategies are not useful and cannot be used correctly.
+As such they have been removed removed them from the list of valid strategies.
```
Comment (victornoel) [inline docs/v0.13.0-release-notes.md]: there is a typo here "have been removed removed them"

File: docs/v0.13.0-release-notes.md [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+The behaviour when the `kubernetes.io/ingress.allow-http` and `ingress.kubernetes.io/force-ssl-redirect` were both specified was somewhat surprising. `ingress.allow-http: false` meant that no routes were not registered for port 80, even if `force-ssl-redirect: true` was set  leading to a 404 where a 3xx upgrade to `https` was expected.
+
+Contour 0.13.0 now prioritizes `force-ssl-redirect`. If this annotation is specified and set to `true`, Contour will always register a port 80 route for the ingress, even if `ingress.allow-http: false`, so that the forced upgrade can take effect.
+
+Fixes #1023 with many thanks to @ceralena.
+
+### `Maglev` and `RingHash` load balancer strategies no longer supported.
+
+`RingHash` and `Maglev` are two balancing / affinity strategies offered by IngressRoute. However due to a lack of understanding of how they worked when they were added in Contour 0.6, neither strategy was properly configured and would only result in random behaviour. 
+Without the ability to configure the hash key, which is usually some for of session af session or cookie, these strategies are not useful and cannot be used correctly.
```
Comment (JayH5) [inline docs/v0.13.0-release-notes.md]: "some for of session af session or cookie" -> "some form of session or cookie"?

File: docs/v0.13.0-release-notes.md (hunk 1 above)
Comment (davecheney) [reply docs/v0.13.0-release-notes.md]: fixed, thank you.

File: docs/v0.13.0-release-notes.md (hunk 2 above)
Comment (davecheney) [reply docs/v0.13.0-release-notes.md]: wow, two big mistakes in the same paragraph. Fixed, thankyou.

**Pull request closed without merging**